examples/run/cpu/seeding_example.cpp

```diff
@@ -54,9 +53,12 @@ int seq_run(const traccc::seeding_input_config& i_cfg,
 
         // Read the detector
         detray::json_geometry_reader<detector_t> geo_reader;
+        detray::detector_builder<typename detector_t::metadata,
```

Comment:
It's been a while since I didn't check the output of seeding examples anymore. yeah so all updated examples may not work here as I intended..

BTW what is the difference between detector reader and detector builder

Reply:
In particular, you can use the builder wihtout the reader (E.g. I plan to switch the detray inbuilt detectors to use a detector builder)

Reply:
Hmm why do you think the current code will not fill any data into `det`?

Reply:
Because ```det``` is not connected to the geometry reader nor the detector builder. You would have to do this:
```
auto det = det_builder.build(host_mr);
```

Reply:
It's in the code tho

Reply:
Oh, OK I must have overlooked that then, sorry! I could not see it when I reviewed. Anyway. the geometry reader is not supposed to be used directly